ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -71,45 +77,25 @@ private[apiserver] final class ApiTransactionService private (
       logging.endInclusive(request.endInclusive),
       logging.parties(request.filter.filtersByParty.keys),
     ) { implicit loggingContext =>
-      val subscriptionId = subscriptionIdCounter.incrementAndGet().toString
-      logger.info(s"Received request for transaction subscription $subscriptionId: $request")
```

Comment:
What was this used for?

Reply:
It pre-dates open-source history. I looked at the closed-source history and I can see that it was used in other log lines in this method, but these have been removed.

Reply:
Cool, thanks!